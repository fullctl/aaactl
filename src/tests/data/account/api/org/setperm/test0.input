{
  "client": "api_client",
  "data": {
    "id": 4,
    "component": "billing.contact",
    "permissions": "crud"
  }
}
