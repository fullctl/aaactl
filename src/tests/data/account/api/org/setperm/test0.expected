{
  "status": 200,
  "response": {"data": [{"org": 2, "orguser": 4, "component": "billing.{org_id}", "permissions": "crud"}], "errors": {}, "profiling": {"queries": 0}}
}
